clients/lsp-json.el

```diff
@@ -99,7 +99,7 @@
                   (list callback))))
 
 (lsp-dependency 'vscode-json-languageserver
-                '(:system "vscode-json-languageserver")
+                '(:system "json-languageserver")
```

Comment:
Is this the correct one?
When I installed this via `npm` as global, the binary name is `vscode-json-languageserver`.
Did Nix change the binary name?

Reply:
The old one seems to be named `json-languageserver`, but that one is obsoleted.
https://www.npmjs.com/package/vscode-json-languageserver

Reply:
I confirmed Nix use the `json-languageserver` name, and I didn't find anything changing the name: https://search.nixos.org/packages?show=nodePackages.vscode-json-languageserver-bin&query=json-language&from=0&size=30&sort=relevance&channel=unstable#disabled

Reply:
Hum, probably have a outdated json-laguageserver, is that right @leungbk ?

Reply:
As it turns out, Nix's version is quite old. I'll have to revert that commit.